boards/olimex-msp430-h2618/include/board-conf.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2014 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#ifndef BOARD_CONF_H
+#define BOARD_CONF_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define INFOMEM     (0x1000)
```

Comment:
I see all msp430 boards have this, but it's not actually used (and also present in the linkerscript) - what is this?